src/index.html

```diff
@@ -1,13 +1,463 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBike</title>
+    <link rel="icon" href="./images/logo.svg">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+            <a href="/" class="logo">
+              <img
+                src="./images/logo.svg"
+                alt="logo MyBike"
+                class="logo__image"
+              >
+            </a>
+
+            <a
+              href="tel:+1 234 5555-55-55"
+              class="icon icon--call"
+            >
+              <div class="icon__tooltip">
+                +1 234 5555-55-55
+              </div>
+            </a>
+
+            <a href="#menu" class="icon icon--menu"></a>
+          </div>
+
+          <h1 class="header__title">
+            Take&nbsp;the&nbsp;Streets
+          </h1>
```

Comment:
I used absolute position for the title because when I only use bottom padding, the title is lower than in the design. Without html entities, the words in the header are placed in 2 lines. I can also use width: max-content but the result will be the same